src/seasidepropertyhandler.h

```diff
@@ -80,6 +82,8 @@ class CONTACTCACHE_EXPORT SeasidePropertyHandler : public QVersitContactHandler
     void detailProcessed(const QContact &, const QContactDetail &detail,
                          const QVersitDocument &, QSet<int> * processedFields, QList<QVersitProperty> * toBeRemoved, QList<QVersitProperty> * toBeAdded);
 
+    static QContactAvatar avatarFromPhotoProperty(const QVersitProperty &property, bool *success);
```

Comment:
I would drop \a success from this interface, the caller can instead test with QContactDetail::isEmpty() on the result.